package.json

```diff
@@ -51,6 +51,7 @@
     "@joystream/types": "0.16.1",
     "@loadable/component": "^5.15.0",
     "@polkadot/extension-dapp": "~0.37.3-17",
+    "@popperjs/core": "^2.10.1",
```

Comment:
I don't think this and `react-popper` are needed for this PR?

Reply:
I removed it from popover-base will wait on it to get merged to rebase discovery here